packages/gafl-webapp-service/src/pages/guidance/accessibility-statement.njk

```diff
@@ -27,49 +27,75 @@
 
         <h2 class="govuk-heading-m">How accessible this website is</h2>
-        <p class="govuk-body">We know some parts of this website are not fully accessible:</p>
+        <p class="govuk-body">All users with access needs can complete the process of buying a fishing rod licence using this service. However we know that some parts of this website are not fully accessible:</p>
         <ul class="govuk-list govuk-list--bullet">
-            <li>The calendar input on the licence start date page is not accessible to those using keyboard input or screen readers. Users with accessibility requirements are able to enter this date using the standard day, month and year inputs.</li>
-            <li>There is a scrolling issue on some older mobile devices that can obscure some content</li>
-            <li>Disabling JavaScript hides the progressive enhancement of the pricing box information during the journey</li>
-            <li>The PDF download table cannot be tabbed through</li>
+            <li>On the licence start date page, the calendar date picker is not accessible to those using keyboards or screen readers. Users with accessibility requirements are able to enter this date using the standard day, month and year fields.</li>
+            <li>The historical calendar dates are too low in contrast</li>
+            <li>The downloadable licence information PDF has a table which cannot be tabbed through</li>
             <li>There is a 3 hour timeout that if reached, will return the user back to the start of the journey</li>
         </ul>
 
-        <h2 class="govuk-heading-m">What to do if you cannot access parts of this website</h2>
-        <p class="govuk-body">If you need information on this website in large print or braille:</p>
+        <h2 class="govuk-heading-m">Feedback and contact information</h2>
+        <p class="govuk-body">If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille:</p>
         <ul class="govuk-list govuk-list--bullet">
             <li>email: <a href="mailto:enquiries@environment-agency.gov.uk">enquiries@environment-agency.gov.uk</a></li>
             <li>call <a href="tel:03708506506">03708 506 506</a> (Monday to Friday, 8am to 6pm)</li>
         </ul>
 
         <h2 class="govuk-heading-m">Reporting accessibility problems with this website</h2>
-        <p class="govuk-body">We are always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we are not meeting the requirements of the accessibility regulations, contact us:</p>
+        <p class="govuk-body">We are always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting  accessibility requirements, contact:</p>
```

Comment:
Did you mean to put the double-space in here?

Reply:
Nope, I copy and pasted from the doc. I'll fix that now, good catch :)